app/src/test/java/org/simple/clinic/patientcontact/PatientContactUiRendererTest.kt

```diff
@@ -107,10 +114,41 @@ class PatientContactUiRendererTest {
     uiRenderer.render(defaultModel().overdueAppointmentLoaded(None))
 
     // then
+    verifyStandardInteractions()
+
     verify(ui).hideCallResultSection()
     verifyNoMoreInteractions(ui)
   }
 
+  @Test
+  fun `if the secure call feature is enabled, show the secure call button and message`() {
+    // when
+    val model = defaultModel(phoneMaskFeatureEnabled = true)
+    uiRenderer.render(model)
+
+    // then
+    verify(ui).showSecureCallButton()
+    verify(ui).showSecureCallMessage()
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `if the secure call feature is disabled, hide the secure call button and message`() {
+    // when
+    val model = defaultModel(phoneMaskFeatureEnabled = false)
+    uiRenderer.render(model)
+
+    // then
+    verify(ui).hideSecureCallButton()
+    verify(ui).hideSecureCallMessage()
```

Comment:
Replace with `verifyStandardInteractions()`

Reply:
In this specific case, I do not want to extract these to a separate method because these are the interactions and assertions that are specific to this test.